google/cloud/firestore_v1/document.py

```diff
@@ -445,7 +449,9 @@ def collections(
             request=request, metadata=self._client._rpc_metadata, **kwargs,
         )
 
+        print("bout to iterate")
         for collection_id in iterator:
+            print(f"collection_id: {collection_id}")
```

Comment:
:)

Reply:
I'm actually fairly started this was committed. I thought I made this change _after_ my last commit. This borderline feels like a glitch in the matrix...